doc/templates.rst

```diff
@@ -782,9 +782,13 @@ The following operators don't fit into any of the other categories:
 
       (1..5)|join(', ')
 
-* ``~``: Converts all operands into strings and concatenates them. ``{{ "Hello
-  " ~ name ~ "!" }}`` would return (assuming ``name`` is ``'John'``) ``Hello
-  John!``.
+* ``~``: Converts all operands into strings and safely concatenates them (to
+  avoid potential XSS). ``{{ "Hello " ~ name ~ "!" }}`` would return 
```

Comment:
the operator is not about *safely* concatenating. It does nothing regarding safety (and so passing it to something assuming it is already safe from XSS would actually creates XSS). The auto-escaping is doing things for safety (what is important to understand is that auto-escaping does not get applied on variables, but on what gets output using `{{ ... }}`, which is why things get escape). the safety does not come from `~`, but from `{{ ... }}` (and the fact that auto-escaping is ON by default)

Reply:
oooh